consensus/consensus_service.go

```diff
@@ -621,13 +615,11 @@ func (consensus *Consensus) NumSignaturesIncludedInBlock(block *types.Block) uin
 
 // getLogger returns logger for consensus contexts added
 func (consensus *Consensus) getLogger() *zerolog.Logger {
-	logOnce.Do(func() {
```

Comment:
revert this change because the data inside the logger will be stale if only init once. It's printing garbage data which confuses debugging.